examples/systemcalls/systemcalls.c

```diff
@@ -16,6 +22,10 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success 
  *   or false() if it returned a failure
 */
+    int status;
+    status = system(cmd);
+    if(status == -1)
```

Comment:
@DivyeshShashikant  think about what would happen if the cmd returned with an error code other than -1 here.

Reply:
are you suggesting that I should have handled other cases as well when the return value is non-zero?